js/ui/grid_core/ui.grid_core.keyboard_navigation.js

```diff
@@ -945,6 +1011,22 @@ var KeyboardNavigationController = core.ViewController.inherit({
         return focusView;
     },
 
+    isRowFocusType: function() {
+        return this.focusType === FOCUS_TYPES.row;
```

Comment:
Why focusType is not string with "row" and "cell" values. This approach for enums is better in js.